sql/sqldb-or-3rdparty.md

```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
 
   Business Continuity Overview
   https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
+
+Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
+
+
+**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
+
+If you have existing SQL Server on-premise installation using AlwaysOn Availability Groups (AG) feature for high-availability and cross-datacenter disaster-recovery, a SQL Server instance installed inside Azure Virtual Machines can join this existing topology as an additional replica. This is not possible for Azure SQLDB. Additionally, if you want to use SQL Server “Log Shipping” DR feature, same restrictions apply.
```

Comment:
I will fix once back from vacation, thanks for catching it.
Regards.
________________________________
Da: David Makogon<mailto:notifications@github.com>
Inviato: ‎24/‎12/‎2015 16:46
A: dmakogon/dbguidance<mailto:dbguidance@noreply.github.com>
Cc: Igor Pagliai<mailto:igorpag@microsoft.com>
Oggetto: Re: [dbguidance] Update sqldb-or-3rdparty.md (#28)


In sql/sqldb-or-3rdparty.md<https://github.com/dmakogon/dbguidance/pull/28#discussion_r48419108>:

> @@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
>
>    Business Continuity Overview
>    https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
> +
> +Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
> +
> +
> +**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
> +
> +If you have existing SQL Server on-premise installation using AlwaysOn Availability Groups (AG) feature for high-availability and cross-datacenter disaster-recovery, a SQL Server instance installed inside Azure Virtual Machines can join this existing topology as an additional replica. This is not possible for Azure SQLDB. Additionally, if you want to use SQL Server “Log Shipping” DR feature, same restrictions apply.


premise -> premises

—
Reply to this email directly or view it on GitHub<https://github.com/dmakogon/dbguidance/pull/28/files#r48419108>.